shared/styles/style-guide.desktop.js

```diff
@@ -124,3 +124,9 @@ export const globalStyles = {
   ...font,
   ...util
 }
+
+export function transition (...properties: Array<string>) : Object {
```

Comment:
just a simple placeholder. need to talk to design about the various transitions we could possible have. doing something simple for now